packages/engine-api-client/src/interface.ts

```diff
@@ -72,6 +71,21 @@ export type BlobsBundle = {
   blobs: Blob[];
 };
 
+export interface IExecutionBuilder {
```

Comment:
Why does the engine-api package declare the builder class interface?

Reply:
Part of unintended changes when resolving merge conflicts. Resolved some in past commits. Missed this. It is removed now.